components/widgets/menu/menu_items/submenu_item.jsx

```diff
@@ -124,30 +150,17 @@ export default class SubMenuItem extends React.PureComponent {
                     onClick={this.onClick}
                 >
                     <span
-                        id={'channelHeaderDropdownIcon_' + id}
-                        className={'fa fa-angle-left SubMenu__icon' + (subMenu && subMenu.length ? '' : '-empty')}
+                        id={'channelHeaderDropdownIconLeft_' + id}
+                        className={'fa fa-angle-left SubMenu__icon-left' + (hasSubmenu && !isMobile ? '' : '-empty' + (isMobile ? ' Mobile' : ''))}
```

Comment:
Why we use a `Mobile` uppercase css class name? we only use uppercase css class names for components.

Reply:
updated to lowercase